e2e/action.yml

```
@@ -13,16 +13,49 @@ runs:
       uses: actions/setup-node@v3
       with:
         node-version: 22
-    - run: npm install
+
+    - name: Set up package-lock.json
+      run: |
+        npm install --package-lock-only --no-audit
```

```suggestion
        npm ci --package-lock-only --no-audit
```

It is really Clean Install. This will never update a package.

---

We do not version `package-lock.file` in our plugin, so this command won't work unfortunately.